crates/bitwarden-wasm-internal/src/platform/mod.rs

```diff
@@ -1,10 +1,28 @@
 use bitwarden_core::Client;
 use bitwarden_vault::{Cipher, Folder};
-use wasm_bindgen::prelude::wasm_bindgen;
+use tsify::{serde_wasm_bindgen, Tsify};
+use wasm_bindgen::prelude::*;
 
 mod repository;
 pub mod token_provider;
 
+#[derive(serde::Serialize, serde::Deserialize, Tsify)]
+#[tsify(into_wasm_abi, from_wasm_abi)]
+pub struct FlagsInput {
+    #[serde(rename = "enableCipherKeyEncryption")]
+    pub enable_cipher_key_encryption: Option<bool>,
+}
+
+impl From<FlagsInput> for std::collections::HashMap<String, bool> {
+    fn from(flags: FlagsInput) -> Self {
+        let js_value = serde_wasm_bindgen::to_value(&flags)
+            .expect("FlagsInput should always serialize successfully");
```

Comment:
We should try to avoid expect, as it would produce a panic. I don't think it would happen in this case but better safe than sorry.
```suggestion
        let js_value = serde_wasm_bindgen::to_value(&flags).unwrap_or_default();
```

Reply:
Good to know! But this should no longer be needed after [9a7ca68](https://github.com/bitwarden/sdk-internal/pull/369/commits/9a7ca686cbacde6fea04679360669ddcb688d930)